contracts/AnyswapV4CallProxy.sol

```diff
@@ -51,40 +51,40 @@ abstract contract MPCManageable {
 
 // support limit operations to whitelist
 abstract contract Whitelistable is MPCManageable {
-    mapping(address => mapping(uint256 => mapping(address => bool))) isInWhitelist;
-    mapping(address => mapping(uint256 =>address[])) whitelists;
+    mapping(address => mapping(uint256 => mapping(address => bool))) public isInWhitelist;
```

Comment:
Although not completely necessary, we should check if changing this to `mapping(bytes32 => bool)` would be cheaper.
The updates to a mapping like that require a key which could be `keccak256(abi.encode(from, chainid, to))`